Wikipedia/Networking/Fetchers/WMFEnglishFeaturedTitleFetcher.m

```diff
@@ -0,0 +1,174 @@
+@end
+
+@interface WMFEnglishFeaturedTitleResponseSerializer : WMFApiJsonResponseSerializer
+@end
+
+@interface WMFTitlePreviewRequestSerializer : AFHTTPRequestSerializer
+@end
+
+@interface WMFEnglishFeaturedTitleFetcher ()
+@property (nonatomic, strong) AFHTTPRequestOperationManager* featuredTitleOperationManager;
+@property (nonatomic, strong) AFHTTPRequestOperationManager* titlePreviewOperationManager;
+@end
+
+@implementation WMFEnglishFeaturedTitleFetcher
+
++ (AFHTTPRequestOperationManager*)featuredTitleOperationManager {
+    AFHTTPRequestOperationManager* featuredTitleOperationManager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
+    featuredTitleOperationManager.requestSerializer  = [WMFEnglishFeaturedTitleRequestSerializer serializer];
+    featuredTitleOperationManager.responseSerializer = [WMFEnglishFeaturedTitleResponseSerializer serializer];
+    return featuredTitleOperationManager;
+}
+
++ (AFHTTPRequestOperationManager*)titlePreviewOperationManager {
+    AFHTTPRequestOperationManager* titlePreviewOperationManager = [AFHTTPRequestOperationManager wmf_createDefaultManager];
+    titlePreviewOperationManager.requestSerializer  = [WMFTitlePreviewRequestSerializer serializer];
+    titlePreviewOperationManager.responseSerializer =
+        [WMFMantleJSONResponseSerializer serializerForValuesInDictionaryOfType:[MWKSearchResult class] fromKeypath:@"query.pages"];
+    return titlePreviewOperationManager;
+}
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        self.featuredTitleOperationManager = [WMFEnglishFeaturedTitleFetcher featuredTitleOperationManager];
+        self.titlePreviewOperationManager  = [WMFEnglishFeaturedTitleFetcher titlePreviewOperationManager];
+    }
+    return self;
+}
+
+- (AnyPromise*)fetchFeaturedArticlePreviewForDate:(NSDate*)date {
+    @weakify(self);
+    MWKSite* site = [MWKSite siteWithLanguage:@"en"];
+    return [self.featuredTitleOperationManager wmf_GETWithSite:site parameters:date]
+           .thenInBackground(^(NSString* title) {
+        @strongify(self);
+        if (!self) {
+            return [AnyPromise promiseWithValue:[NSError cancelledError]];
+        }
+        return [self.titlePreviewOperationManager wmf_GETWithSite:site parameters:title]
+        .then(^(NSArray<MWKSearchResult*>* featuredTitlePreviews) {
+            return featuredTitlePreviews.firstObject;
+        });
+    });
+}
+
+@end
+
+@implementation WMFEnglishFeaturedTitleRequestSerializer
+
++ (NSDateFormatter*)featuredArticleDateFormatter {
+    static NSDateFormatter* feedItemDateFormatter;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        feedItemDateFormatter = [[NSDateFormatter alloc] init];
+        feedItemDateFormatter.dateFormat = @"MMMM d, YYYY";
+        // feed format uses US dates—specifically month names
+        feedItemDateFormatter.locale = [NSLocale localeWithLocaleIdentifier:@"en-US"];
```

Comment:
@bgerstle I left this but it got lost in an outdated diff…

I don't think en-US is a locale? This is probably evaluating to nil. en_US is a locale. Maybe it accepts the hyphen format too? Not sure, but would check.

If it is evaluating to nil, then do we even need to set the locale?

Also any reason not to use the currentLocale? Even if they are getting content in english, chrome is usually still in the users device language.